### ISSUE TITLE
how to train on other losses? when i change to triplet loss, it just pops up with no warning,why?.

### ISSUE THREAD

**Issue by feixiaoyu2223** [nwojke/cosine_metric_learning#38]

**Comment by nwojke** [comment]
Please provide a detailed description of the problem.

**Comment by feixiaoyu2223** [comment]
Later, i modified loss function and it can run, but it feels that there is not much difference after changing the loss function?


------------------




 




------------------ 原始邮件 ------------------
发件人: "Nicolai Wojke"<notifications@github.com>;
发送时间: 2019年5月29日(星期三) 下午4:38
收件人: "nwojke/cosine_metric_learning"<cosine_metric_learning@noreply.github.com>;
抄送: "Fred.Fly"<441275397@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [nwojke/cosine_metric_learning] how to train on other losses?when i change to triplet loss, it just pops up with no warning,why?. (#38)




Please provide a detailed description of the problem.

**Comment by nwojke** [comment]
We found the difference in performance similar in our experiments as well (see paper).
